### PR TITLE
riak_test for Riak Enterprise 1.4 Replication v3 proxy_get

### PR DESCRIPTION
**Pull request by bookshelfdave**
code around "object_three" has been commented out, as I need a better way to detect when proxy_get is enabled/disabled. If anyone would be so kind as to review and file any issues against this PR, I would be more than happy to revisit after vacation.

Best of luck on your release!

Cheers -
Dave

### CODE REVIEW

**Comment by bookshelfdave** [comment]
oops, I left an extra timer:sleep() in there that I'll need to remove, but the test should pass as-is.

File: riak_test/tests/repl_v3_test.erl [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+        [proplists:get_value(key, O) || O <- proplists:get_value(contents,
+                erlcloud_s3:list_objects(?TEST_BUCKET, U1C2Config))]),
+
+    lager:info("check that the 2 other objects can't be read"),
+    %% We expect errors here since proxy_get will fail due to the
+    %% clusters being disconnected.
+    ?assertError({aws_error, _}, erlcloud_s3:get_object(?TEST_BUCKET,
+            "object_two", U1C2Config)),
+%    ?assertError({aws_error, _}, erlcloud_s3:get_object(?TEST_BUCKET,
+%            "object_three", U1C2Config)),
```
Comment (reiddraper) [inline riak_test/tests/repl_v3_test.erl:156]: what're these commented lines for?

**Comment by reiddraper** [comment]
Looks good to me except for the above question.

File: riak_test/tests/repl_v3_test.erl (hunk 1 above)
Comment (bookshelfdave) [inline riak_test/tests/repl_v3_test.erl:156]: The code around the get for "object_three" has been commented out, as I need a better way to detect when proxy_get is enabled/disabled. I'll try to resolve this in the next couple days. Apologies for the delay.

**Comment by bookshelfdave** [comment]
pushing off until next week, but close to the top of my list. I can close in the meantime if you like.

**Comment by reiddraper** [comment]
no need

**Comment by kuenishi** [comment]
Note: after #848 merged, pushing any more commits without rebasing to develop  or running CI tests will break the plt file in our CI environment. Please rebase to develop, or branch including #848 .

**Pull request closed without merging**